### PR TITLE
Fix readme/docs and add changelist

### PR DESCRIPTION
**Pull request by qiaojialin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -177,108 +103,54 @@ The server can be stopped with ctrl-C or the following script:
...
 |1970-01-01T08:00:00.001+08:00|               101|
 +-----------------------------+------------------+
 Total line number = 1
-It costs xxxs
-```
 
-If your session looks similar to what's above, congrats, your IoTDB is operational!
-
-For more on what commands are supported by IoTDB SQL, see our documentation [Chapter 5: IoTDB SQL Documentation](https://iotdb.apache.org/#/Documents/0.8.0/chap5/sec1).
+IoTDB> quit/exit
```
Comment (jt2594838) [inline README.md]: This line seems unclear, better to be
> If you want to exit the client, use:
> IoTDB > quit
> or
> IoTDB > exit

File: docs/Documentation/QuickStart.md [hunk 2]
```diff
@@ -55,306 +56,142 @@ IoTDB provides you two installation methods, you can refer to the following sugg
...
-IoTDB> INSERT INTO root.ln.wf01.wt01(timestamp,status,temperature) values(200,false,20.71)
execute successfully.
-```
-The data we've just inserted displays like this:
+# Unix/OS X
+> tools/export-csv.sh -h <ip> -p <port> -u <username> -pw <password> -td <xxx.csv> [-tf <time-format>]
 
+# Windows
+> tools\export-csv.bat -h <ip> -p <port> -u <username> -pw <password> -td <xxx.csv> [-tf <time-format>]
 ```
```
Comment (jt2594838) [inline docs/Documentation/QuickStart.md]: I think it is better to explain the parameters, especially the last one. Or you may link to the detailed user guide.

File: README.md (hunk 1 above)
Comment (qiaojialin) [reply README.md]: fixed

File: docs/Documentation/QuickStart.md (hunk 2 above)
Comment (qiaojialin) [reply docs/Documentation/QuickStart.md]: removed

**Pull request merged**